src/util/BaseLoginController.js

```diff
@@ -104,6 +105,16 @@ define(['okta', 'q'], function (Okta, Q) {
       this.trigger('pageRendered', {page: this.className});
 
       this.trigger('afterRender', { controller: this.className });
+
+      this.listenTo(this, 'transactionError', function (err) {
```

Comment:
can't remember exactly, so can you check if this happens after we "normalize" some errors with the logic in https://github.com/okta/okta-signin-widget/blob/4e0c5d2823eb6a283f8e6f216eabf7e6eeba1eca/src/util/Util.js#L23?

Reply:
Great question - we modify the `xhr` object by replacing the the following:

```javascript
 xhr.responseJSON = {
  errorSummary: 'something'
};
```

Normalization occurs on the request response, so the `err` object received here is already transformed.

Reply:
curious your thoughts on why built another layer to handle the error (trigger `transactionError` at line 58 and handles it at line 109) instead of invoke the handler at line 58 directly?

Reply:
When I was originally testing this change, we needed to emit the `afterError` event **after** some code execution path. Turns out, we were firing this event a little too early.

I've pushed a new commit with changes to address your concern above. Please see [27e63a4](https://github.com/okta/okta-signin-widget/pull/594/commits/27e63a4abe99b398f1384dc6e680c994ee3df87b)